data_conversion.py

```diff
@@ -3,54 +3,48 @@
 
 @author: qinyuzhu
 """
-import flirt
+
 import pandas as pd
 from datetime import datetime, timedelta
 
-df = pd.read_csv("test_conversion.csv")
-print(df)
+def type_conversion():
+     """Function that do data type conversion for time features
```

Comment:
I think you are still working on this function. I don't think we are stating all the information needed here.